compiler/p/runtime/PPCArrayCopy.inc

```diff
@@ -63,14 +63,14 @@
    .globl    __forwardQuadWordArrayCopy_vsx{DS}
    .globl    .__quadWordArrayCopy_vsx
    .globl    __quadWordArrayCopy_vsx{DS}
-   .globl    .__forwardLEArrayCopy
-   .globl    __forwardLEArrayCopy{DS}
-   .globl    .__leArrayCopy
-   .globl    __leArrayCopy{DS}
    .globl    .__postP10ForwardCopy
    .globl    __postP10ForwardCopy{DS}
    .globl    .__postP10GenericCopy
    .globl    __postP10GenericCopy{DS}
+   .globl    .__forwardLEArrayCopy
+   .globl    __forwardLEArrayCopy{DS}
+   .globl    .__leArrayCopy
+   .globl    __leArrayCopy{DS}
```

Comment:
Can you please clarify why these (and other declarations dealing with these helpers) are being moved?

Reply:
my interim version of the code already removed all code related to leArrayCopy.  but, when i need to make this fix, i cannot remove them (because OpenJ9 side still refers to the symbols). So that, I added the leArrayCopy stuffs back in for the time being (but only the names and labels though, all real code related to leArrayCopy not added back in).

Reply:
In that case, I think it would be best to remove all changes involving these two helpers from this PR. They're not really related to the problem at hand and the helpers themselves can't be fully removed until OpenJ9 references to them are removed anyway. Once the OpenJ9 references to these symbols have been removed, a PR properly removing these from OMR can be opened.

Reply:
Given this is causing a build break in a downstream project and this cleanup is minor I'm going to go ahead and merge this and we can clean up the helper labels once the downstream dependencies are removed.